app/helpers/http_helper.rb

```diff
@@ -0,0 +1,47 @@
+require "net/http"
+
+module HttpHelper
+  def self.post(url, body)
```

Comment:
If we assume this always returns a JSON... should we encode it the name somewhere? 
Alternatively is not commit to return a JSON below.